diff --git a/node_modules/@payhere/payhere-mobilesdk-reactnative/android/build.gradle b/node_modules/@payhere/payhere-mobilesdk-reactnative/android/build.gradle
index fd1e440..629497c 100644
--- a/node_modules/@payhere/payhere-mobilesdk-reactnative/android/build.gradle
+++ b/node_modules/@payhere/payhere-mobilesdk-reactnative/android/build.gradle
@@ -45,6 +45,7 @@ apply plugin: 'com.android.library'
 // apply plugin: 'maven'
 
 android {
+    namespace "lk.payhere.mobilesdk"
     compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
     buildToolsVersion safeExtGet('buildToolsVersion', DEFAULT_BUILD_TOOLS_VERSION)
     defaultConfig {
